src/createServer.js

```diff
@@ -1,13 +1,24 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const cors = require('cors');
+  const { router: usersRouter } = require('./routes/users.js');
+  const { router: expensesRouter } = require('./routes/expenses.js');
+  const { expensesService } = require('./services/expensesService.js');
+  const { userService } = require('./services/usersService.js');
```

Comment:
Put `requires` on top level, unless it is necessary to do otherwise. Doen't really change anything, requires are cached anyway.